jenkins/Dockerfile/release/debug-image/ticdc

```diff
@@ -1,4 +1,4 @@
-FROM pingcap/alpine-glibc:alpine-3.14.6
+FROM rockylinux:9.0
 RUN apk add --no-cache tzdata bash curl socat
```

Comment:
O, `apk` is only for alpine. rocky linux is a RedHat like Linux. please use`yum` or `dnf` package manage tools.